plan/plan.go

```diff
@@ -181,12 +181,6 @@ func shouldUpdateProviderSpecific(desired, current *endpoint.Endpoint) bool {
 		return false
 	}
 	for _, c := range current.ProviderSpecific {
-		// don't consider target health when detecting changes
```

Comment:
This shouldn't be dropped, https://github.com/kubernetes-sigs/external-dns/pull/877